presto-hive/src/main/java/com/facebook/presto/hive/s3/HiveS3Config.java

```diff
@@ -374,4 +375,16 @@ public HiveS3Config setS3UserAgentPrefix(String s3UserAgentPrefix)
         this.s3UserAgentPrefix = s3UserAgentPrefix;
         return this;
     }
+
+    public boolean isSkipGlacierObjects()
+    {
+        return skipGlacierObjects;
+    }
+
+    @Config("hive.s3.skip-glacier-objects")
```

Comment:
Please add this new config to the docs (`hive.rst` file, `Amazon S3 Configuration` section).